app/package.json

```diff
@@ -60,10 +60,14 @@
     "@bcgov-cas/sso-react": "^1.0.1",
     "@bcgov/bc-sans": "^1.0.1",
     "@button-inc/bcgov-theme": "^1.0.1",
+    "@emotion/react": "^11.7.0",
+    "@emotion/styled": "^11.6.0",
```

Comment:
where is this used? We already have two different styling libraries (`styled-jsx` and `styled-components`)

Reply:
They came with @mui I can remove them and see if it affects the Autocomplete

Reply:
mui Autocomplete explodes without those dependencies

Reply:
I see emotion can be replaced by styled-components, but it's currently discouraged as there is an issue with styled-components, SSR and MUI: https://mui.com/guides/styled-engine/#how-to-switch-to-styled-components